### PR TITLE
lpv - Long Period Variables

### PR DESCRIPTION
**Pull request by AshishMahabal**
Added a golden set for HR diagram, period for a folded light curve, and some basic information.
More will have to be added later.

### CODE REVIEW

**Review by dmitryduev — CHANGES_REQUESTED** [review]
Awesome, thanks @AshishMahabal! 
Please remove all png files (git rm --cached doc/data/*png), and taxonomy.html (git rm --cached doc/_static/taxonomy.html) -- these files get generated automatically.

File: doc/field_guide__lpv.md
Comment (dmitryduev) [inline doc/field_guide__lpv.md:28]: You are saying above that it is tens of days - ...

**Review by dmitryduev — APPROVED** [review]
👍🏻

File: doc/field_guide__lpv.md
Comment (AshishMahabal) [reply doc/field_guide__lpv.md:28]: I feel tens of days is consistent with hundred. May be several tens of days. Will tighten up the wording when I make more changes.

File: doc/field_guide__lpv.md
Comment (przemekmroz) [inline doc/field_guide__lpv.md:17]: Can you describe particular subtypes? What are L stars? I believe RV Tau stars belong to Type II Cepheids rather than "long period variables". (Type II Cepheids and LPVs follow different period-luminosity relations.) Long-period variables: https://ui.adsabs.harvard.edu/abs/2009AcA....59..239S/abstract, type II cepheids: https://ui.adsabs.harvard.edu/abs/2008AcA....58..293S/abstract

Comment (AshishMahabal) [reply doc/field_guide__lpv.md]: I have updated the subtypes. And added the refs. I think we need to add a few figures to bring out the differences between the subtypes.

Comment (przemekmroz) [reply doc/field_guide__lpv.md]: Thanks Ashish. I think OSARGs also belong to "long-period variables" (basically, as a star evolves along the giant branch, the amplitude of pulsations grows, you start with low-amplitude oscillations (OSARGs), then the pulsation amplitude grows (SRVs --> Miras). It will be helpful to add a light curve of an SRV variable.

Comment (AshishMahabal) [reply doc/field_guide__lpv.md]: Przemek, I have shifted OSARGs to LPVs with the comment that not everyone agrees.

Comment (przemekmroz) [reply doc/field_guide__lpv.md]: Thanks. Can you add an example light curve of a semiregular variable?

Comment (AshishMahabal) [reply doc/field_guide__lpv.md]: Added one (for the first pair of coordinates that you gave).

**Comment by dmitryduev** [comment]
@przemekmroz @AshishMahabal @janvanroestel is this one ready to be merged?

**Comment by AshishMahabal** [comment]
I can merge once either of Przemek or Jan approve.

-ashish

**Comment by dmitryduev** [comment]
Let's wait until EDR3 is ingested into Gloria and #26 is merged, otherwise the docs will not get updated on scope.ztf.dev (the corresponding GH action will fail).

**Comment by AshishMahabal** [comment]
In that case we could do it with the Ra Dec plot.

Ashish

On May 5, 2021 6:54 PM, Dmitry Duev ***@***.***> wrote:

Let's wait until EDR3 is ingested into Gloria and #26<https://github.com/ZwickyTransientFacility/scope/pull/26> is merged, otherwise the docs will not get updated on scope.ztf.dev (the corresponding GH action will fail).

**Review by przemekmroz — APPROVED** [review]

**Pull request merged**
